### DISCUSSION TITLE
如何让nonebot直接返回消息而不调用api

### DISCUSSION THREAD

**Discussion by doudoucao** [nonebot/discussions#206] [问答]
如果不想调用各个平台的api，而是让nonebot作为server，直接response消息，怎样实现好一点？

**Comment by yanyongyu** [comment]
什么叫直接response消息，`matcher.send`/`bot.send`回复消息？

**Reply by doudoucao** [reply]
bot.send呀，就是不想调call_api，而是直接fastapi response

**Reply by yanyongyu** [reply]
那你可能需要自己写适配器，只有适配器可以直接操作回复，这就和nb插件没有关系了
